<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortBox</class>
 <widget class="QGroupBox" name="SortBox">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>163</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>242</width>
    <height>163</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>242</width>
    <height>163</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Сортировка</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="2" column="0">
    <widget class="QRadioButton" name="todayRB">
     <property name="text">
      <string>Только сегодня</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">alphabetsort</string>
     </attribute>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QRadioButton" name="weekRB">
     <property name="text">
      <string>Следующая неделя</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">alphabetsort</string>
     </attribute>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QRadioButton" name="monthRB">
     <property name="text">
      <string>Следующий месяц</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">alphabetsort</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="allRB">
     <property name="text">
      <string>Всё</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">alphabetsort</string>
     </attribute>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="okB">
     <property name="text">
      <string>Ок</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QRadioButton" name="AZcheck">
     <property name="text">
      <string>А-Я</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">datesort</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QRadioButton" name="ZAcheck">
     <property name="text">
      <string>Я-А</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">datesort</string>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="alphabetsort"/>
  <buttongroup name="datesort"/>
 </buttongroups>
</ui>
